apisix/schema_def.lua

```diff
@@ -78,6 +78,11 @@ local health_checker = {
                 timeout = {type = "number", default = 1},
                 concurrency = {type = "integer", default = 10},
                 host = host_def,
+                port = {
+                    type = "integer",
+                    minimum = 1000,
```

Comment:
I think the minimum value should be `1`.

Reply:
I think that in general, ports below 1000 are open to the system, so here is set to 1000.

Reply:
for example the HTTP port `80` is fine.

Reply:
OK, I will update.